### ISSUE TITLE
Cant package 4.27

### ISSUE THREAD

**Issue by egidijus1231** [Sixze/ALS-Refactored#41]
The game works fine in editor, but when I try to package it it gives me this error:
![image](https://user-images.githubusercontent.com/11089377/142743375-2a7e1970-7d6a-41ff-8df3-58a9e63b87d4.png)

**Comment by Sixze** [comment]
Weird, find `DefaultALS.ini`, remove all text inside and try again, this should help.

**Comment by egidijus1231** [comment]
I removed all text inside DefaultALS.ini and now it packages project, but when I press PLAY it just crashes.
Also in editor it also crashes. I guess it still requires that content from ini file.

![Screenshot_11](https://user-images.githubusercontent.com/11089377/142760439-9d3bfc65-f1d5-481d-99b6-1e70d464a039.jpg)

When I use DefaultALS.ini from github it just give me this, but maybe its because of my file structure?
With "DefaultALS.ini" it works in Editor, but not in Packaged.
I have my project called ALS_Ref. It is empty C++ project, that contains plugins folder and the plugin itself is in the:
ALS_Ref/Plugins/ALS/ALS.uplugin. I used same structure when I tested ALS Community c++ version and it worked ok.

![image](https://user-images.githubusercontent.com/11089377/142760326-665a1422-6683-4d6d-be57-8d36bbf11627.png)

Thank you
